fiftyone/utils/torch.py

```
@@ -517,6 +637,44 @@ def __init__(self, d):
+            use_numpy=self.use_numpy,
+            force_rgb=True,
+        )
+        if self.transform is not None:
+            img = self.transform(img)
+
+        if self.raw_inputs:
+            return img
+
+        else:
+            if self.using_half_precision:
+                imgs = imgs.half()
```

Ruff (0.8.2)

Local variable `imgs` is assigned to but never used

Remove assignment to unused variable `imgs`

(F841)